src/TwigComponent/src/DependencyInjection/TwigComponentExtension.php

```diff
@@ -68,7 +79,7 @@ class_exists(AbstractArgument::class) ? new AbstractArgument(sprintf('Added in %
             ])
         ;
 
-        $container->setAlias(ComponentRendererInterface::class, 'ux.twig_component.component_renderer');
+        $container->setAlias(ComponentTemplateFinderInterface::class, 'ux.twig_component.component_template_finder');
```

Comment:
I think this isn't needed. This isn't something I'd expect users to need to autowire.